### DISCUSSION TITLE
[Development] Create User Account Locally After Waitlisted

### DISCUSSION THREAD

**Discussion by msteinbrick** [shared-recruiting-co/shared-recruiting-co#178] [Q&A] [answered]
Hi, quick question on getting the app working locally:
- I've successfully signed in via Gmail, filled out the sign up form, and now my user account is waitlisted
- How do I now activate the account? I can't figure out what action to trigger getting the account off the waitlist

Ran into this while working on https://github.com/shared-recruiting-co/shared-recruiting-co/issues/177

**Comment by devstein** [comment, marked as answer]
Great question

1. Connect to the local Supabase Postgres instance
```bash
psql 'postgresql://postgres:postgres@localhost:54322/postgres
```

2.  Run this SQL command to allow your email to create an account
```sql
update waitlist set can_create_account = true where email = 'YOUR_EMAIL';
```

3. Refresh the page when you are on the waiting list success screen or try click the Account button in the header



I can add to the documentation. Let me know what other unforeseen issues you run into getting your environment setup

**Comment by msteinbrick** [comment]
@devstein - thanks, that got me one step further. 

Now, I'm having an issue after I accept the terms of service and try to connect to Gmail: 
![Screenshot 2023-03-23 at 9 46 40 AM](https://user-images.githubusercontent.com/95698968/227275613-7cb22ddf-c111-45cd-9035-c38e256023f1.png)

Seems like the Google Cloud Console needs the localhost endpoint as an approved redirect URI? I couldn't figure out a work around

**Reply by devstein** [reply]
Thanks for sharing a screenshot. This has to do with your OAuth app setup in GCloud. See screenshots below from my setup. The important parts are:

1. Add OAuth redirect URIs and allowed origins 
2. Add you email(s) as a test user

<img width="830" alt="Screenshot 2023-03-23 at 10 09 51 AM" src="https://user-images.githubusercontent.com/7771049/227282279-9aac0c94-9fd6-448b-91ac-eddb3695636d.png">
<img width="775" alt="Screenshot 2023-03-23 at 10 08 05 AM" src="https://user-images.githubusercontent.com/7771049/227282320-432c470c-4f51-452a-91f7-aed1c48169f8.png">

**Comment by msteinbrick** [comment]
@devstein thanks, for some reason I was thinking it had to work on the production (the powering https://sharedrecruiting.co/) Google Cloud Project 🤦
